### DISCUSSION TITLE
Domestic and International Price

### DISCUSSION THREAD

**Discussion by barykada90** [etsy/open-api#977] [Q&A]
There seems to be something missing in the entire API's schema.
We have a system where we need to synchronise quantity - in order to do that, we need to use PUT updateListingInventory call (as that's the only endpoint allowing to update quantity) but then the price is required which is **international price**.
After the update, if the listing has domestic&international pricing enabled, the call makes it off and the domestic price field blank.

Please help as this makes synchronisation completely pointless if some fields need to be rewritten and lost.

**Comment by hyyyper** [comment]
I have encountered this as well.
This options is only available for shops in some regions (ex: India) but it's not mentioned anywhere in the API.
Any of those shops cannot use the API to manage inventory without sacrificing the domestic & international pricing.

**Reply by barykada90** [reply]
That's just ridiculous, you can't use it then without messing up the customer's data...

Can you give some more insights into:
>  This options is only available for shops in some regions (ex: India) but it's not mentioned anywhere in the API.

**Comment by KushGoyal** [comment]
This is super frustrating. Data of one my client got messed up.

The response from the inventory API has price as a json object with currency code. But when posting the inventory data the price field is a float.

This is not the first time etsy API has caused us huge trouble. Please get this fixed etsy team!!

**Comment by barykada90** [comment]
Do we know if there'll be any progress on this? it's kinda makes updating and synchronising everything redundant for listings with domestic and international prices...

**Reply by hyyyper** [reply]
My understanding is that the Etsy team does not review these discussions. Have you found any workarounds that allowed you to modify the international prices or does it still remain out of our reach?

**Reply by barykada90** [reply]
Then what's the point of having this forum if they're not checking them? Very frustrating :<
I'm afraid I haven't found any workaround... :(

**Reply by barykada90** [reply]
@nateswart - I don't want to believe that's true? :)

**Comment by Gareth064** [comment]
> Then what's the point of having this forum if they're not checking them? Very frustrating :< I'm afraid I haven't found any workaround... :(

Because it is a "Communtiy" discussion board now.

When the repo was first created, Etsy used it to work with the community to build\Test the v3 API. But the moment the API when Live, they jumped ship, turned off raising issues, and said this is only for community members to help each other. Etsy Devs, like @nateswart will pop in every now and then to let us know if a breaking change is inbound, or apologies for a last minute breaking change.

The only way to "try" to talk to an Etsy developer is via deceloper@etsy.com but even then they are not very respondant.

Unfortunately the Etsy API, a long with a rich app echo system, isn't a priority for Etsy. They simply want users using their Website and App. Which is frustrating for people like us which want to build apps which plug missing functionality from Etsy to help sellers work more efficiently etc.

**Comment by alexh-swdev** [comment]
Hi,
so, by any chance, are there any updates on this?

We just learned we have the same problem: When updating the quantities, only the international price is left in the offer. When pulling the inventories (GET getListingInventory) we also only get one offering with the international price instead of (expected) two.

Is there any way to find out, (via tha API) whether a product as only one price or an domestic/international? Then it this major(!) issue could be at least work-arounded by a notification to check the products manually.

**Reply by datenfalke** [reply]
Hello,
we have the same major problem now - through API i can set the "Everywhere else" price but not the most important prices for Germany and USA. We have over 3000 products listet which makes it impossible to change them all manually :-(
I would be happy to be able to reset all prices to only one single price - but why it is not possible to set all prices through API, that should be possible for a worldwide company. In the moment we are just unable to change prices at all, very unhappy paying Etsy customer :'(

**Comment by boly38** [comment]
Hi, 

thanks for this thread 👍 👍 , 

This seems obvious, but to avoid confusion there is a typo in mail provided previously by [Gareth064](https://github.com/Gareth064) : 
-> its [developer@etsy.com](mailto:developer@etsy.com) (source: last etsy email communication about "Etsy Open API Personalization Migration" - Feb 6 2026)

I just send a feature request to this email with an example and referencing this discussion too. I hope Etsy team will at least answer me or here to provide their vision about this issue. 🤞

Best Regards

**Comment by noegruoc** [comment]
**Still broken in 2026 — Domestic & Global Pricing makes inventory/variations unmanageable via the API**

  Confirming this is still unresolved after the **Jan 2026 GA** of Domestic & Global Pricing, and it's now hitting many more shops (Etsy enables it regionally, sometimes by default).

  **Reproducible behaviour (verified live, May 2026):**
  - `PUT /v3/application/listings/{id}/inventory` with the **same** product/offering structure → **200 OK**.
  - The **same** PUT with **one added variation value** (new structure) → **400**:
    `"There was a problem with / : To edit variations for listings with domestic & global pricing, open Etsy.com in a mobile or desktop browser."`
  - So the block is specifically on **structural variation changes** (add/remove option), not on all inventory writes.

  **Read side is also broken (data integrity):**
  - `getListing` / `getListingInventory` expose a **single `price`** per offering. The domestic / global / US-specific tiers are **not returned** anywhere (`buyer_price` is null server-side). A tool literally
  cannot see the international price it's about to overwrite.
  - There is **no field** indicating a listing uses Domestic & Global Pricing, so tools cannot even detect/flag affected listings to warn the seller.

  **Concrete asks (in priority order):**
  1. Return all price tiers (domestic / global / US) per offering in `getListingInventory`.
  2. Allow `updateListingInventory` (or a new param/endpoint) to **add/edit variation structure while preserving** the regional price tiers — at minimum, adding an option without forcing the feature off.
  3. Add a boolean flag (e.g. `has_domestic_global_pricing`) to `getListing` so integrations can detect it.

  Impact: every third-party tool + every seller integration is **locked out of variation management** on these listings, silently and irreversibly via API. This is a growing share of listings since the GA.
  Please escalate to the inventory/pricing API owners.

**Reply by cyrilchandelier** [reply]
I have emailed developer@etsy.com last week on that topic and linked to this comment as it encapsulate well what we all need to correctly manage inventory with domestic/regional pricing enabled.

**Reply by datenfalke** [reply]
I wrote also to developer@etsy.com, perhaps more can do it.

**Reply by maccarib** [reply]
I also did
